### PR TITLE
Issue #4227: Image fields: Do not allow min resolution to be larger than max resolution

### LINKED ISSUE #4227: Image fields: Do not allow min resolution to be larger than max resolution
Filing this as a follow-up to #4221.

When creating image fields, there is no validation for the resolution when min > max. So you end up with this:

69861930-80763100-12ed-11ea-813c-f39916cb9f6b

---
~~PR by @klonos https://github.com/backdrop/backdrop/pull/3009~~
PR by docwilmot https://github.com/backdrop/backdrop/pull/3109

### PR DESCRIPTION
**Pull request by docwilmot**
Fixes https://github.com/backdrop/backdrop-issues/issues/4227
Replaces https://github.com/backdrop/backdrop/pull/3009

### CODE REVIEW

File: core/modules/image/tests/image.test [hunk 1]
```diff
@@ -872,6 +872,32 @@ class ImageFieldDisplayTestCase extends ImageFieldTestCase {
       '%max' => $schema['fields'][$field_name .'_title']['length'],
       '%length' => $test_size,
     )));
+
+    // Check that an image minimum size cant be larger than its maximum size.
```
Comment (jenlampton) [inline core/modules/image/tests/image.test]: can we change `Check that an image minimum size cant be larger than its maximum size.` into `Check that image minimum size is not larger than maximum size.`?

Comment (docwilmot) [reply core/modules/image/tests/image.test]: Resolved

File: core/modules/image/image.field.inc [hunk 2]
```diff
@@ -200,6 +203,26 @@ function image_field_instance_settings_form($field, $instance) {
   return $form;
 }
 
+/**
+ * #element_validate handler for image_field_instance_settings_form().
```
Comment (jenlampton) [inline core/modules/image/image.field.inc]: If you're still nearby, can we also change this to `Element validate` instead of `#element_validate`?

Comment (docwilmot) [reply core/modules/image/image.field.inc]: Done

**Comment by quicksketch** [comment]
Merged into 1.x and 1.15.x. Thanks!

**Pull request merged**